### ISSUE TITLE
Process return void results

### ISSUE THREAD

**Issue by loretoparisi** [primaryobjects/lda#14]
When working with huge documents (~400), each of them like 500b-1K I often get a `[ [], [], [] ]` result from the `process` method.
What I do is to read the files (one document per line), and parse as an array and pass as sentences to the `lda`. What I have found is that for a small number of documents (between 1 to 50) it works, then the output becomes void like  `[ [], [], [] ]`.

**Comment by primaryobjects** [comment]
If you're able to debug this further and issue a PR, I'd be happy to take a look. Thank you! :)

**Comment by loretoparisi** [comment]
@primaryobjects ok but do you have any idea how what could happen in the function `process` for this data size? I get no errors, not sure if there is any trapped exception some where...If you point me in the right way I will be happy to debug it :)

For 200 sentences (vocabulary:2105 documents:200) I get some void results like:

```
[ [ { term: 'loved', probability: 0.12 },
    { term: 'give', probability: 0.035 },
    { term: 'girl', probability: 0.025 },
    { term: 'dance', probability: 0.022 },
    { term: 'body', probability: 0.022 } ],
  [],
  [] ]
```

while for 100 sentences (vocabulary:1837 documents:100) I get 

```
[ [ { term: 'don', probability: 0.049 },
    { term: 'man', probability: 0.037 } ],
  [ { term: 'loved', probability: 0.023 } ],
  [ { term: 'loved', probability: 0.13 },
    { term: 'give', probability: 0.022 },
    { term: 'falling', probability: 0.022 },
    { term: 'body', probability: 0.022 } ] ]
```

For 250 and more sentences (vocabulary:2105 documents:250) I get void `[ [], [], [] ]`.
I can see in this case that here:
```
things.sort().reverse();
          console.log(things);
          if(topTerms>vocab.length) topTerms=vocab.length;
```

the `things` objects is filled with values like

```
[
'9.7e-7_remind_remind',
  '9.6e-7_wind_wind',
  '9.6e-7_wexford_wexford',
  '9.6e-7_water_water',
  '9.6e-7_van_van',
  '9.6e-7_todai_today',
],
[
...
]
```

I'm using these parameters:

```
alphaValue:0.1
betaValue:.01
numberOfTopics: 3,
numberOfTermsPerTopic: 5
```

and I can see some default values in

```
var V = vocab.length;
      var M = documents.length;
      var K = parseInt(numberOfTopics);
      var alpha = alphaValue || 0.1;  // per-document distributions over topics
      var beta = betaValue || .01;  // per-topic distributions over words
      documents = documents.filter((doc) => { return doc.length }); // filter empty documents

      lda.configure(documents,V,10000, 2000, 100, 10, randomSeed);
      lda.gibbs(K, alpha, beta);
```

**Comment by loretoparisi** [comment]
@primaryobjects I did some more tests. I have used instead of the PorterStemmer you provide different ones like the Porter and Lancaster stemmers provided by [NaturalNode](https://github.com/NaturalNode/natural), but that was not the problem.

I have also used a direct mapping i.e. a lemmatization file by languages available [here](http://www.lexiconista.com/datasets/lemmatization/), but this does not seems to be the problem.
